### PR TITLE
Terriann/feature/form ui - Issue #17

### PR DESCRIPTION
**Pull request by terriann**
Added a form template in a new file `form.html` and wired up the popup's button to load that instead of the options page.

The options page is also attached to the Chrome app as an options page for the app and I couldn't get the button to load the content of the `formMenu` directory because it's outside of the extension (and I didn't want to loose any of the proof of concept or tests in those files.

The page is setup for us to write in the JS object and iterate the card element depending on the number of counts.

@jakedurell if there are fields you would like to see updated to a different type (button group, dropdown etc.) let me know; I opted to keep everything simple input fields for the most flexibility upfront.

### CODE REVIEW

**Comment by terriann** [comment]
Whoops - got my issue # wrong in my commit messages 🤦‍♀ 
If y'all want me to recommit with the correct issue just let me know - it's no issue (pun intended)

**Comment by jakedurell** [comment]
I have not looked at the form in Browser, but the HTML looks like it covers everything. Maybe create a separate CSS eventually?

I know there was a lot of back and forth, but yesterday it seems we landed on host the file locally with an express server. So the form menu folder can live outside of the extension directory. @drewandrew ?

Also, I think we should call it "formMenu" so as to not confuse it with other individual forms?

**Comment by jakedurell** [comment]
image

That's beautiful!

**Comment by terriann** [comment]
Thank you! I'm very glad you like it - it's nice when things come out just the way I saw it in my head.

I can definitely rename the file but I'm afraid I don't quite see how to get this to load from the other directory so I'm not sure how to test if I get it setup in the right place. @drewandrew maybe that's something you can help me with?

I'll move the CSS into a separate stylesheet in a new `/css/` directory and commit those changes to the branch too.

File: extensionDirectory/popup.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ let docketInfo = document.getElementById('docketInfo');
 
 createPetition.onclick = function (element) {
     chrome.tabs.create({
-        url: chrome.extension.getURL('options.html#window')
+        url: chrome.extension.getURL('formMenu.html')
```
Comment (drewandrew) [inline extensionDirectory/popup.js]: Changing this to `url: 'http://localhost:3000' should do it! Just need to swap in your form.html for the index.html in the formMenu/ directory ... :crossed_fingers: it's that easy

**Comment by terriann** [comment]
I merged the latest changes that are on the master branch and added an "add new count" button because I'd remembered we'd talked about the user needing the ability to add more counts.

image

**Pull request merged**